Library/Formula/ffmpegthumbnailer.rb

```diff
@@ -12,16 +11,52 @@ class Ffmpegthumbnailer < Formula
     sha1 "052f2227429e215db559dcbddb2cdca838111d59" => :lion
   end
 
-  revision 2
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "ffmpeg"
 
-  depends_on 'pkg-config' => :build
-  depends_on 'jpeg'
-  depends_on 'libpng'
-  depends_on 'ffmpeg'
+  patch :DATA
 
   def install
-    system "./configure", "--disable-debug", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
-    system "make install"
+    # The CMake script does not allow fine-grained control of man directories or
+    # other installation paths beneath the prefix
+    inreplace "CMakeLists.txt", "${CMAKE_INSTALL_PREFIX}/man/man1", man1
```

Comment:
I have a feeling they used some kind of template or generator to create their CMakeLists.txt, then never looked at it again except to bump the version number. So yeah, there is a lot of room for improvement in there, this just being the tip of the iceberg (or the mouth of the rabbit hole).